rllib/env/wrappers/pettingzoo_env.py

```diff
@@ -154,7 +154,7 @@ def close(self):
         self.env.close()
 
     def render(self):
-        return self.env.render(self.render_mode)
+        return self.env.render()
```

Comment:
in the newest petting zoo version, are render modes no longer configurable? 

if so the whole pr looks p much looks good to me